.github/workflows/build.yml

```
@@ -0,0 +1,53 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'corretto'
+        java-version: '8'
```

Not for this PR, but a reminder to myself that we should move this to Java 11.

---

Ah, this comment reminded me that I've already done this https://github.com/guardian/elastic-search-monitor/pull/27! So I think we want this to match here @Mark-McCracken 

```suggestion
        java-version: '11'
```